File: MonteCarlo1.m
%% The given code estimates the value of pi. It uses Monte Carlo Technique by estimating area of a sector enclosed in a square.%%
countIN = 0;
countOUT = 1000;

for i=1:countOUT
x(i) = rand;
y(i) = rand;

 dist = sqrt(x(i)^2+ y(i)^2);
 if (dist <= 1)
    countIN = countIN + 1;
 end
end


Ratio = 4*(countIN/countOUT) ;
    
Error = pi - Ratio
  plot(x,y,'o')

File: MonteCarlo1.py
import numpy as np
import math as mat

count = 90000;
countin,x,y = 0,0,0;

for i in range(count):
 x = np.random.rand();
 y = np.random.rand();
 dist = pow(x,2) + pow(y,2);
 if dist  <= 1:
     countin = countin +1;
ratio = 4*(countin/count);


print('ratio :' + str(ratio))
print('count :' + str(count))
print('countin :'  + str(countin))



